spyder/plugins/debugger/widgets/main_widget.py

```diff
@@ -162,8 +207,14 @@ def setup(self):
 
         # Main toolbar
         main_toolbar = self.get_main_toolbar()
-        for item in [search_action, enter_debug_action,
-                     inspect_action]:
+        for item in [search_action,
+                     inspect_action,
+                     enter_debug_action,
+                     next_action,
+                     continue_action,
+                     step_action,
+                     return_action,
+                     stop_action]:
```

Comment:
```suggestion
        for item in [next_action,
                     continue_action,
                     step_action,
                     return_action,
                     stop_action,
                     enter_debug_action,
                     inspect_action,
                     search_action]:
```

I think it's better to show the buttons that control the debugger first.